packages/mask/src/components/DataSource/useActivatedUI.ts

```diff
@@ -18,6 +18,7 @@ export function useCurrentVisitingIdentity() {
 export function useCurrentIdentity(): {
     identifier: ProfileIdentifier
     linkedPersona?: { nickname?: string; identifier: PersonaIdentifier; fingerprint?: string }
+    fingerprint?: string
```

Comment:
Remove this. Use `linkedPersona?.identifier.compressedPublicKey` instead.

Reply:
done